app/assets/javascripts/.eslintrc.js

```diff
@@ -25,5 +25,7 @@ module.exports = {
     initializeAllFollowButts: false,
     initializeSponsorshipVisibility: false,
     ActiveXObject: false,
+    sendHapticMessage: false,
+    showModal: false,
```

Comment:
FYI: you can also declare these: /* global sendHapticMessage, showModal */ at the top of the file, since lots of function are global anyway in this part of the code

Reply:
Ah, I did read that while going through the eslint documentation for globals, I figured configuring it in globals in eslintrc is the convention compared to disabling it file by file.

Reply:
I think either way is fine :)